wave_matching/src/icp.cpp

```diff
@@ -13,17 +13,20 @@ ICPMatcher::ICPMatcher(float res) {
     } else {
         this->resolution = -1;
     }
-    wave::ConfigParser parser;
+    ConfigParser parser;
```

Comment:
can we also make another constructor that takes in a subset of the important params, instead of the full config file.  In some cases we may have multiple instantiations of this object with different parameters, and I can see it getting annoying to have different config files for each object, or use the one default config file then manually call the "set" methods.